forc-plugins/forc-tx/src/lib.rs

```diff
@@ -624,8 +622,11 @@ impl TryFrom<Create> for fuel_tx::Create {
             .map(|s| fuel_tx::Witness::from(s.as_bytes()))
             .collect();
         let create = fuel_tx::Transaction::create(
-            create.gas.price,
-            create.gas.limit,
+            create.gas.price.unwrap_or(0),
+            create
+                .gas
+                .limit
+                .unwrap_or(fuel_tx::ConsensusParameters::DEFAULT.max_gas_per_tx),
```

Comment:
Should we be using the defaults from the node here as well?

Reply:
Actually it seems like forc-tx doesn't specify the node, so there's no way to use the node's defaults